var/spack/repos/builtin/packages/shtools/package.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Shtools(Package):
```

Comment:
Technically this should be a MakefilePackage, but then it will look a little more complicated (makeopts will need to be its own function, and you'll override both `build` and `install`, or `build_targets` and `install_targets`).